chain/store/snapshot.go

```diff
@@ -197,7 +197,7 @@ func recurseLinks(ctx context.Context, bs bstore.Blockstore, walked *cid.Set, ro
 			rerr = err
 		}
 	})
-	if err != nil {
+	if err != nil && err != io.EOF {
```

Comment:
This is a new bug in cbor-gen.

Fixed in 37c43cae8e145618a0ca6f2c8c1e3e5ef39a7d40 / https://github.com/whyrusleeping/cbor-gen/pull/63